lib/polyn/application.rb

```diff
@@ -72,10 +74,10 @@ def initialize(options)
     # @param payload [Hash] the message to publish
     def publish(type, payload)
       event = Event.new({
-        type:   type,
-        source: "#{source_prefix}.#{name}",
-        data:   payload,
-      })
+                          type:   type,
+                          source: "#{source_prefix}.#{name}",
+                          data:   payload,
+                        })
 
```

Comment:
I do not like this styling change. It doesn't change anything. But I am not a fan.